crowbar_framework/app/assets/stylesheets/installer.scss

```diff
@@ -0,0 +1,9 @@
+.installer {
+  li {
+    padding-right: 5px;
+    line-height: 2em; border-radius: 3px;
+  }
+  .action > li >  span { margin-right: 5px; margin-left: 15px }
```

Comment:
Properties should be ordered margin-left, margin-right<br>Selector should have depth of applicability no greater than 2, but was 4<br>Declaration should be terminated by a semicolon